app/helpers/application_helper/toolbar/middleware_deployments_center.rb

```diff
@@ -19,4 +20,34 @@ class ApplicationHelper::Toolbar::MiddlewareDeploymentsCenter < ApplicationHelpe
       ]
     ),
   ])
+  button_group('middleware_deployments_operations', [
+    select(
+      :middleware_deployment_deploy_choice,
+      'fa fa-play-circle-o fa-lg',
+      t = N_('Operations'),
+      t,
+      :enabled => false,
+      :onwhen  => "1+",
+      :items => [
+        button(
+          :middleware_deployment_redeploy,
+          'pficon pficon-restart fa-lg',
+          N_('Redeploy these #{ui_lookup(:table=>"middleware_deployments")}'),
+          N_('Redeploy archive'),
+          :url_parms => "main_div",
+          :enabled => false,
+          :onwhen  => "1+",
+          :confirm => N_("Do you want to redeploy these deployments ?")),
+        button(
+          :middleware_deployment_undeploy,
+          'fa fa-eject fa-lg',
+          N_('Undeploy these #{ui_lookup(:table=>"middleware_deployments")}'),
+          N_('Undeploy archive'),
```

Comment:
In Hawkular UI we use only the action name, so just "Redeploy" or "Undeploy". In fact there's no "Undeploy" it is called "Disable", which stops the deployment and "Remove" which actually removes the deployment (and undeploys (disables) it if deployed (enabled)). See screenshot:

![image](https://cloud.githubusercontent.com/assets/1737954/16045712/9c1f1948-3241-11e6-967c-3e6565265aaa.png)

Should we go to such level of splitting the actions ?